### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by cavis**
Upgrades! Docker! Everything!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,35 +21,73 @@ Triggered via SNS notifications (see [Announce](https://github.com/PRX/announce)
...
-the `TEST_BUCKET` and `DESTINATION_BUCKET` defined in `config/test.env`.  Then, just...
+SQS callbacks contain the following JSON data:
 
-```
-npm install
-npm test # or npm run watch
-```
+| Key    | Description |
+| ------ | ----------- |
+| id     | ID of the Image that triggered this job
```
Comment (farski) [inline README.md:30]: Is that an ID from CMS?

File: buildspec.yml [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
+      - "cp env-example .env"
+      - "echo AWS_ACCESS_KEY_ID=$CMS_ACCESS_KEY_ID >> .env"
+      - "echo AWS_DEFAULT_REGION=$CMS_DEFAULT_REGION >> .env"
+      - "echo AWS_SECRET_ACCESS_KEY=$CMS_SECRET_ACCESS_KEY >> .env"
+      - "echo CODECOV_TOKEN=$CODECOV_TOKEN >> .env"
+      - "echo CODEBUILD_BUILD_ID=$CODEBUILD_BUILD_ID >> .env"
+      - "echo PRX_REPO=$PRX_REPO >> .env"
+      - "echo PRX_COMMIT=$PRX_COMMIT >> .env"
+      - "echo PRX_BRANCH=$PRX_BRANCH >> .env"
+      - "echo PRX_GITHUB_PR=$PRX_GITHUB_PR >> .env"
```
Comment (farski) [inline buildspec.yml:25]: Why do all the echos instead of just listing the vars in the docker-compose file?

**Review by farski — COMMENTED** [review]
Wasn't familiar with whatever a `callback` is, but otherwise looked pretty good.

File: README.md (hunk 1 above)
Comment (cavis) [reply README.md:30]: Yes.

File: buildspec.yml (hunk 2 above)
Comment (cavis) [reply buildspec.yml:25]: I actually need the ENVs set inside the docker container, inside the other docker container.

So if I list/set them here, they end up defined in the host container, but not the child container (running the tests).

**Comment by cavis** [comment]
The "callback" is just the name of the SQS queue that CMS watches.  "production_cms_image_callbacks" or something.

File: package.json [hunk 3]
```diff
@@ -19,19 +22,18 @@
...
     "follow-redirects": "^1.2.3",
     "q": "^1.4.1",
-    "sharp": "^0.17.2"
+    "sharp": "^0.21.0"
   },
   "devDependencies": {
+    "aws-sdk": "^2.4.7",
     "chai": "^3.5.0",
-    "codecov": "^1.0.1",
+    "codecov": "PRX/codecov-node",
```
Comment (kookster) [inline package.json:32]: we have our own?

Comment (kookster) [reply package.json:32]: Looked it up - makes sense to get it to work with ci

**Review by kookster — APPROVED** [review]
lgtm

**Pull request merged**